File: Backend/Service/Models/Configuration/ConfigurationModel.cs
using Service.Models.Classifier;

namespace Service.Models.Configuration
{
    public class ConfigurationModel
    {
        public  int Id { get;  set; }
        public string Code { get; set; }
        public int PlanId { get; set; }
        public int? PettyCashId { get; set; }
        public int? WarehouseId { get; set; }
        public bool? PaymentTypeCash { get; set; }
        public bool? PaymentTypeQr { get; set; }
        public bool? PaymentTypeCard { get; set; }
        public bool? IsActive { get; set; }
        public BaseClassifierModel Plan { get; set; }
    }
}

File: front-end/src/app/modules/configuration/models/configuration.model.ts
import { AuthUserConfiguration } from "app/modules/auth/models/authUser.configuration.model";
import { Classifier } from "app/modules/warehouse/models/warehouseProduct";

export interface Configuration {
    id: number | null;
    code: string;
    paymentTypeCash:  boolean | null;
    paymentTypeQr:  boolean | null;
    paymentTypeCard:  boolean | null;
    warehouseId :  number | null;
    pettyCashId :  number | null;
    planId :  number | null;
    plan : Classifier;
    isActive: boolean | null;
    authUserConfiguration : AuthUserConfiguration | null;
  }
  